src/modes/direct_select.js

```diff
@@ -73,6 +79,21 @@ module.exports = function(ctx, opts) {
     selectedCoordPaths = [about.coord_path];
   };
 
+  function pathsToCoordinates(paths) {
+    return paths.map(coord_path => feature.getCoordinate(coord_path));
+  }
+
+  function coordinatesToPoints(coordinates) {
+    return coordinates.map(coords => ({
+      type: Constants.geojsonTypes.FEATURE,
+      properties: {},
```

Comment:
Let's store the coord_path and the feature_id here. This will let users be able to take this feature and take actions on the selected coordinates external to Draw if they please.